usb/backend/libusb1.py

```diff
@@ -634,6 +647,16 @@ def __init__(self, dev):
         self.devid = dev.devid
         _check(_lib.libusb_open(self.devid, byref(self.handle)))
 
+class _FileDescriptorDeviceHandle(object):
+    def __init__(self, fd, ctx):
```

Comment:
I find it rather interesting and like having others for feedback. But I'm short on time, so changes will not happen for at least two weeks, maybe more. And you might not be able to change code or suggest actual changes?

As you are deep into the whole material and I was thinking of renaming my pull branch because I started with `master` ... maybe you just open another Pull request and reference mine, while I will close but keep the history?

https://stackoverflow.com/a/48480353/9360161

Reply:
It looks like the primary location for getting feedback is the [mailing list](https://sourceforge.net/projects/pyusb/lists/pyusb-users). It doesn't look it has any mention of termux in the archives..

Nothing is urgent. For now I have something working with just using a local branch with some changes. I don't think I'll have any time in the next week or so to poke at it again.

Reply:
Ok.
I'm in the same situation as I also have a collection of scripts to patch/extend it for my use-cases...